plugins/modules/cloud/misc/terraform.py

```diff
@@ -33,6 +33,19 @@
         vars.tf/main.tf/etc to use.
     type: path
     required: true
+  plugin_paths:
+    description:
+      - List of paths containing Terraform plugin executable files.
+      - Plugin executables can be downloaded from U(https://releases.hashicorp.com/).
+      - When set, the plugin discovery and auto-download behavior of Terraform is disabled.
+      - The directory structure in the plugin path can be tricky. The Terraform docs
+        U(https://learn.hashicorp.com/tutorials/terraform/automate-terraform#pre-installed-plugins)
+        show a simple directory of files, but in testing that does not work, the directory structure
```

Comment:
I'm not sure if "in testing that does not work" is a good sentence here, maybe "in action" or "actually"?

Reply:
Went with "actually"